runtime/pangolin/src/lib.rs

```diff
@@ -154,6 +158,13 @@ frame_support::construct_runtime! {
 		Ethereum: pallet_ethereum = 36,
 		Evm: pallet_evm = 37,
 		MessageTransact: darwinia_message_transact = 38,
+
+		// Pangolin <> Pangoro
+		BridgeMoonbaseGrandpa: pallet_bridge_grandpa::<Instance1> = 39,
```

Comment:
Why `Moonbase`?

The Pangoro and Pangolin under the Rococo relay chain if I understand correctly?

Reply:
Pangoro2 attach to Moonbase relay

Reply:
You are right. I got it